outputs.tf

```diff
@@ -1,14 +1,7 @@
-output "id" {
-  description = "ID of the created example"
-  value       = module.this.enabled ? module.this.id : null
+output "service_account_name" {
+  value = local.service_account_name
 }
 
-output "example" {
-  description = "Example output"
-  value       = module.this.enabled ? local.example : null
-}
-
-output "random" {
-  description = "Stable random number for this example"
-  value       = module.this.enabled ? join("", random_integer.example[*].result) : null
+output "namespace" {
+  value = local.service_account_name
```

Comment:
Stylistically this is confusing even if correct. I would create a new local and set it equal to the service account name or vise verse.